packages/p2p/src/pages/buy-sell/currency-selector/currency-selector.scss

```
@@ -1,4 +1,4 @@
-.buy-sell .currency-selector {
+.buy-sell-currency-selector {
```

wouldn't it be better just to remove the `.buy-sell` then we can keep the styles name to its component name

```suggestion
.currency-selector {
```

---

I have renamed it to this since this component is only used on buy-sell page. The component name has been changed as well in refactoring buy-sell page.